### PR TITLE
update docker base

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

TBS

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: dockers/nvidia/Dockerfile [hunk 1]
```diff
@@ -12,52 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM nvcr.io/nvidia/cuda:11.1.1-runtime-ubuntu20.04
+# https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/rel_21-03.html#rel_21-03
+FROM nvcr.io/nvidia/pytorch:20.10-py3
```
Comment (awaelchli) [inline dockers/nvidia/Dockerfile:16]: isn't it better to use LTS version? :)

**Review by awaelchli — APPROVED** [review]

File: dockers/nvidia/Dockerfile (hunk 1 above)
Comment (Borda) [reply dockers/nvidia/Dockerfile:16]: NGC does not have LTS tag, this shall be Ubuntu 18.04

**Review by carmocca — APPROVED** [review]

**Review by ethanwharris — APPROVED** [review]
Working for me :smiley:

**Review by justusschock — APPROVED** [review]

File: requirements/examples.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 torchvision>=0.5
 gym>=0.17.0
+ipython[all]
```
Comment (kaushikb11) [inline requirements/examples.txt:3]: It seems to be getting installed in the Docker image as well?

Comment (ethanwharris) [reply requirements/examples.txt:3]: @kaushikb11 Yeah, this was a hack to get it to work with PL 1.2.7 - the docker image grabs old requirements and so doesn't necessarily install `ipython[all]` (which is needed for it to work), so has to be manually installed in the `Dockerfile` too.

Comment (Borda) [reply requirements/examples.txt:3]: anyway we have it docker as if-needed so no reinstall :]

**Review by tchaton — APPROVED** [review]
LGTM !

**Pull request merged**